analyzeme/src/profiling_data.rs

```diff
@@ -86,7 +87,7 @@ impl ProfilingData {
         Ok(ProfilingData { event_decoder })
     }
 
-    pub fn metadata(&self) -> &Metadata {
+    pub fn metadata(&self) -> Metadata {
```

Comment:
This method is called in a few expensive loops, it seems. Let's cache the `Metadata` value in a [OnceCell](https://doc.rust-lang.org/std/cell/struct.OnceCell.html) so we don't copy the cmd string a million times.

Reply:
Done in d348f3ace236a84e130e53145e859385220308de